README.md

```diff
@@ -5,17 +5,21 @@ Maintain the source code for GraphQL related tools.
+
+Every GraphQL service defines a set of types that completely describe the set of possible data you can query on that service. Then, when queries come in, they are validated and executed against that schema. GraphQL schemas for a service are now most often specified using what’s known as the `GraphQL SDL` (schema definition language), also sometimes referred to as just GraphQL schema language. Ballerina GraphQL tooling will make it easy for you to start the development of a client in Ballerina for a given GraphQL SDL and a GraphQL document configured in a GraphQL config file by generating Ballerina client skeletons. You can generate a single client in Ballerina for multiple GraphQL documents for a given GraphQL SDL. It also enables you to generate multiple Ballerina modules for multiple GraphQL projects to work with different GraphQL APIs. In addition, the GraphQL tooling supports GraphQL SDL schema generation for a given Ballerina GraphQL service. The user will be able to generate the SDL schema and export it to a specific directory for a selected set of services in the given bal file.
 
 The Ballerina GraphQL tooling support provides the following capabilities.
- 
- 1. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL schema (SDL) and a GraphQL document using the CLI tool.
 
- 2. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL SDL and multiple GraphQL documents using the CLI tool.
+1. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL schema (SDL) and a GraphQL document using the CLI tool.
+
+2. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL SDL and multiple GraphQL documents using the CLI tool.
+
+3. Generating multiple Ballerina modules from a given GraphQL config file configured with multiple GraphQL projects. Each project will generate a separate Ballerina module. This enables you to work with multiple GraphQL APIs by configuring each GraphQL API under a separate project.
+
+4. Generate the GraphQl SDL schema for a given Ballerina GraphQL service(s).
+
+### Command for Ballerina GraphQL client generation
 
- 3. Generating multiple Ballerina modules from a given GraphQL config file configured with multiple GraphQL projects. Each project will generate a separate Ballerina module. This enables you to work with multiple GraphQL APIs by configuring each GraphQL API under a separate project.
-    
 The `graphql` command in Ballerina is used for GraphQL to Ballerina code generations. It supports several usages in the Ballerina GraphQL tool as follows.
```

Comment:
This is now invalid. The tool can be used to `GraphQL to Ballerina` and `Ballerina to GraphQL` as well.